ibm_catalog.json

```diff
@@ -53,6 +53,35 @@
           "index": 1,
           "install_type": "fullstack",
           "working_directory": "solutions/fully-configurable",
+          "dependencies": [
+            {
+              "flavors": [
+                "fully-configurable"
+              ],
+              "id": "4322cf44-2289-49aa-a719-dd79e39b14dc-global",
+              "name": "deploy-arch-ibm-scc-workload-protection",
+              "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
+              "input_mapping": [
+                {
+                  "dependency_input": "region",
+                  "version_input": "region",
+                  "reference_version": true
+                },
+                {
+                  "dependency_input": "prefix",
+                  "version_input": "prefix",
+                  "reference_version": true
+                },
+                {
+                  "dependency_input": "existing_resource_group_name",
+                  "version_input": "existing_resource_group_name",
+                  "reference_version": true
+                }
+              ],
+              "version": "v1.10.1",
```

Comment:
Are you supposed to pinpoint the version? I know we were doing this for Core PaaS (and surround too), but not sure if that's applicable to ICD modules too.

Reply:
In order to catch such mismatched version we should add the addons testing in this same PR. See Key Protect or Observability tests for reference